docker/brew.Dockerfile

```diff
@@ -181,12 +175,11 @@ COPY docker/install-dynamic-plugins.py ./
 RUN chmod a+r ./install-dynamic-plugins.py
 
 # Copy embedded dynamic plugins
-COPY --from=builder $CONTAINER_SOURCE/dynamic-plugins/ ./dynamic-plugins/
+COPY --from=builder $CONTAINER_SOURCE/dynamic-plugins/dist ./dynamic-plugins/dist
```

Comment:
if we are only copying from builder's `dynamic-plugins/dist` folder, why do we need to `find dynamic-plugins -type f -not -name 'dist' -delete` on line 117 ?